lib/edit/OfflineEditBasic.js

```diff
@@ -488,7 +488,19 @@ define([
 
                     // we need to identify ADDs before sending them to the server
                     // we assign temporary ids (using negative numbers to distinguish them from real ids)
-                    layer._nextTempId = -1;
+                    // query the database first to find any existing offline adds, and find the next lowest integer to start with.
+                    this._editStore.getNextLowestTempId(layer, function(value, status){
+                        if(status === "success"){
+                            console.log("_nextTempId:", value);
+                            layer._nextTempId = value;
+                        }
+                        else{
```

Comment:
When looking at the `else` statement here, I think the only time `status != success` is when there is an error, correct?

If that's true, do we need the `else` statement? 

Also, we could handle errors but that's gonna be messy, messy code that may be overkill.

Thoughts?